apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -92,9 +50,12 @@ export class AuthorizationService {
 		return this.authorizationHelper.hasOneOfPermissions(user, requiredPermissions);
 	}
 
+	// TODO: think about it
+	// TODO: Rename
 	public async getUserWithPermissions(userId: EntityId): Promise<User> {
-		const userWithPermissions = await this.loader.getUserWithPermissions(userId);
+		// replace with service method getUserWithPermissions
```

Comment:
Das meint, statt dem UserRepo den UserService zu benutzen, oder? Da hatte ich mich nämlich gerade gewundert, dass du das Repo injected hast.

Reply:
Noch gibt es kein Service / bzw. getUserWithPermissions Method. Das wäre das follow up ticket. Da würde ich aber gerne warten was bei dem me Ticket bei raus kommt und ob man Teile davon gleich nachnutzen könnte. 

Die Änderung wird aber auch noch mal heftig. Da es den verwendeten User in fast allen Services verändert. 

Ich könnte den Kommentar auch erstmal weg lassen, da es das Ticket schon gibt.

Reply:
Oder du erklärst das im Kommentar und verlinkst das Ticket.

Reply:
Ticket hinzugefügt